cypress/component/annotation-explanation.cy.js

```diff
@@ -1,32 +1,104 @@
 import annotExplanation from "~/components/annot-explanation";
 
+const props = { activeCategory: "Age" };
+let state, store;
+
 describe("explanation", () => {
 
-    it("starts collapsed", () => {
-        cy.mount(annotExplanation);
+    // Setup
+    beforeEach(() => {
+
+        state = {
+
+            categories: {
+
+                "Age": {
+
+                    explanation: "Age explanation"
+                }
+            }
+        };
+        store = {
+
+            getters: {
+
+                getExplanation: () => (p_category) => {
+
+                    return state.categories[p_category].explanation;
+                }
+            },
+            state: state
```

Comment:
did you mean to put both `getters` and `state` under the `store` object?

Reply:
@surchs Yes, I did. Why? Is there a reason not to?

Reply:
>if the getExplanation getter would handle the "is explanation missing" by returning null if no explantion exists, then you could still provide a default value inside of the component but wouldn't have to worry about potentially missing keys - and you could also remove the mapState thing for categories.

@surchs I'm having trouble understanding this comment. I think maybe there are two different ideas being confused here re: default value via props and default text for the UI. Can you break it down/reword? Or maybe pseudocode it here?

Reply:
no, I think adding them both under `store` is the way to go. sorry for not being clear: I asked because atm you are **not** organizing them both under the `store` object but then pass the `state` reference inside the `store` object. This looked odd to me so I wanted to know if you had meant to combine them both here to simplify the mock.

Reply:
@surchs Scratch that. I think I understand your comment now.

If the getter in the store `getExplanation` returns null, I can check for that in `explanationText` in the component and provide the default text there - thus eliminating the need to use `mapState` for `categories`.

Good suggestion! I will apply it.

Also, afterwards I'm going to rename `annotation-explanation.cy.js` to `annot-explanation.cy.js` to keep the file name in line with other annotation page component tests.

Reply:
neat! and good catch on the name!